### PR TITLE
New PR for the product bundle support - does work now ;-)

### PR DESCRIPTION
**Pull request by wpichler**

### CODE REVIEW

**Comment by wpichler** [comment]
@flotho What are you next steps ? Please let me know to avoid duplicate work. I do plan to work on the product export side now...

**Comment by flotho** [comment]
hi @wpichler , thnaks for the update,
from now the product exportation is working correctly from my point of view.
what is it missing from your point of view?

**Comment by flotho** [comment]
Hi @wpichler ,

I have som issues while importing templates, it failed on the product category importer. Did you already had this kind of issues?
Moreover and maybe the most important, what is your idea for importing the product.template? do you start directly with product.template import or do you first start product.product and afterwards start the product.template.

thanks for your answer

**Comment by wpichler** [comment]
Hi @flotho 

product template does currently do not check dependency for product.category - i have to add it (added a TODO for now). So you have to update product categories first.

The basic idea for product template import is:
The normal product.product import does not import products which are variants from magento any more (it does check this with filter visibility flag - variants are not visible !)
The product.template import does import the template - and then it does import the linked variants - so you get a complete product template in odoo

File: connector_magento/models/product_template/common.py [hunk 1]
```diff
@@ -398,40 +398,18 @@ def escape(term):
             res = self._call('configurable-products/%s/children' % (escape(sku)), None)
             return res
 
-    def get_product_datas(self, data, id=None, saveOptions=True):
```
Comment (flotho) [inline connector_magento/models/product_template/common.py:402]: could you explain what will be the replacement of this part ?
we need minimum datas while exporting templates and products.

**Comment by flotho** [comment]
Hi @wpichler ,
I'm also about to start to work on the exportation, 
do you have any updates before we coudl share ?
best regards

**Comment by wpichler** [comment]
Hi @flotho 

perfekt - my current pr is up to date - i've had some other work to do ;-)

**Pull request merged**